teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super()
+    @specialization = specialization
+  end
```

Comment:
- [ ] The constructor should have all the parameters of the parent class and with specialization as an extra parameter. Please add all parameters in the initialize method and add parameters of the super class in the super method as well.